hack/testing/logging.sh

```diff
@@ -488,6 +488,9 @@ function reinstall() {
   os::cmd::try_until_text "oc get pods -l component=kibana" "Running" "$(( 3 * TIME_MIN ))"
   os::cmd::try_until_text "oc get pods -l component=curator" "Running" "$(( 3 * TIME_MIN ))"
   os::cmd::try_until_text "oc get pods -l component=fluentd" "Running" "$(( 3 * TIME_MIN ))"
+
+  # wait 3 minutes to let Fluentd process some records to send to ES
+  sleep 180
```

Comment:
This seems excessive - if it is really taking this long, we need to find out why